NEWS

```diff
@@ -9,6 +9,7 @@ Current git version
 
   * Client window titles (controlled by the theme attributes 'title_height',
     'title_color', 'title_font')
+  * New command 'help' for online help on the object tree
```

Comment:
This sounds like a browser will be opened with some online help :( Is it a typo of *inline*?

Reply:
Hm right, maybe 'live documentation' is better (instead of 'online help').

Reply:
Much better, thanks!